app/src/server/api/external/v1Api.router.ts

```diff
@@ -75,12 +46,8 @@ export const v1ApiRouter = createOpenApiRouter({
       },
     })
     .input(
-      // TODO: replace this whole mess with just `chatCompletionInput` once
-      // no one is using the `reqPayload` field anymore.
+      // TODO: replace this whole mess with just `chatCompletionInputReqPayload`
       z.object({
```

Comment:
Have we checked no one is using the old sdk versions? I'm not sure what Axis is on for example.

Reply:
I searched the db for recent request logs that contained reqPayload and came up empty. We deprecated this field 3 months ago, and I think most of our users have updated since then.